docs/docker-image-details.md

```diff
@@ -82,23 +83,22 @@ Changelog
 
 Changelog
 
-- trino:435-gravitino-0.5.1
-  - Based on Gravitino 0.5.1, you can know more information from 0.5.1 release notes.
+- apache/gravitino-playground:trino-435-gravitino-0.6.0-incubating (Switch to Apache official DockerHub repository)
+  - Use `datastrato/trino:435-gravitino-0.5.1` Dockerfile to rebuild the image.
```

Comment:
Is this comment right?

Reply:
Yes, I will modify it as all the docker images are based on the release-0.6 NOT 0.5.1